### PR TITLE
OSDOCS-4906: Adding oc-mirror OCI GA to RNs

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-4906

Link to docs preview:
https://58933--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-installation-oc-mirror-oci-ga

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@sherine-k @zhouying7780 Can you please review?

Preview: https://58933--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-installation-oc-mirror-oci-ga

**Review by sjhala-ccs — COMMENTED** [review]
LGTM

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -190,11 +190,33 @@ For more information about scaling a user-provisioned cluster by using the BMO,
 * `ap-northeast-2` (Seoul)
 * `ap-southeast-7` (Bangkok)
 
+[id="ocp-4-13-installation-oc-mirror-oci-ga"]
+==== Using the oc-mirror plugin to include local OCI Operator catalogs is now generally available
+
+You can now use the oc-mirror plugin to mirror local OCI Operator catalogs on disk to a mirror registry. This feature was previously introduced as a Technology Preview in {product-title} 4.12 and is now generally available in {product-title} 4.13.
+
+This release introduces support for the following features when using the OCI feature:
```
Comment (sherine-k) [inline release_notes/ocp-4-13-release-notes.adoc:198]:
```suggestion
This release introduces support for the following features when local oci catalogs are included:
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -190,11 +190,33 @@ For more information about scaling a user-provisioned cluster by using the BMO,
...
+
+* Pruning images from the target mirror registry
+* Incremental mirroring to only mirror what has changed since the last time you ran the tool
+* Namespace hierarchy for alternative names for catalogs in the target mirror registry
+
+[IMPORTANT]
+====
+* If you used the Technology Preview OCI feature for the oc-mirror plugin for OpenShift Container Platform 4.12, you can no longer use the OCI feature of the oc-mirror plugin to create an image set archive as a first step to mirroring to a fully disconnected cluster.
```
Comment (sherine-k) [inline release_notes/ocp-4-13-release-notes.adoc:206]:
```suggestion
* If you used the Technology Preview OCI local catalogs feature for the oc-mirror plugin for OpenShift Container Platform 4.12, you can no longer use the OCI feature of the oc-mirror plugin to copy the catalog image locally as a first step to mirroring to a fully disconnected cluster.
```

**Review by sherine-k — CHANGES_REQUESTED** [review]
Hi @bergerhoffer 
I spotted just a few nits... nothing major. 
Thanks so much

**Review by sherine-k — APPROVED** [review]
Thank you so much @bergerhoffer

**Pull request merged**